os/arch/arm/src/tiva/lm3s_ethernet.c

```diff
@@ -797,8 +806,8 @@ static void tiva_receive(struct tiva_driver_s *priv)
 			 */
 
 			EMAC_STAT(priv, rx_ip);
-			arp_ipin(&priv->ld_dev);
-			ipv4_input(&priv->ld_dev);
+			//arp_ipin(&priv->ld_dev);
```

Comment:
There are many commented codes. If they are not needed, Let's remove all of them.